### ISSUE TITLE
notttttt working 

### ISSUE THREAD

**Issue by kishanj918** [kanji-keraliya/RNPlaidExample#2]
then why do you create

**Comment by kanji-keraliya** [comment]
@kishanj918 Now project is working fine in Android as well as in iOS

**Comment by kishanj918** [comment]
and did you able to redirect back in app as well?

**Comment by kanji-keraliya** [comment]
For iOS its working fine but in android application crashed on system back (not close button). and I already created issue in plaid for that

I have created this project to find out plaid issue integration issue and shared with Plaid team for investigation.

**Comment by kishanj918** [comment]
Yes, the same issue for both unable to redirect back after the transaction in webview (Ireland banks) and for native banks working fine, have you received any news regarding the issue?

**Comment by kanji-keraliya** [comment]
No, I didn’t received any updates from plaid
